docs/ems_reporting.md

```diff
@@ -0,0 +1,27 @@
+# EMS reporting
+
+Cumulus reports usage statistics to the [ESDIS Metrics System (EMS)](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems).
+Two types of reports are generated: ingest and distribution.
+
+## Ingest
+
+TODO - add content here
+
+## Distribution
```

Comment:
Should probably add something about the config that goes in the app/config.yml for this

Reply:
Once we moved the log location and reports target to the internal bucket, there were no changes to `app/config.yml`.